app/scenes/CollectionNew.tsx

```diff
@@ -30,7 +30,7 @@ class CollectionNew extends React.Component<Props> {
   icon = "";
 
   @observable
-  color = "#4E5C6E";
+  color = Collection.randomColor();
```

Comment:
Same comment here as sever side, `randomElement(colorPalette)` would be totally fine and just avoid some undocumented indirection

Reply:
Fixed!